### ISSUE TITLE
Save episode progress and resume playback

### ISSUE THREAD

**Issue by keineahnungking** [asbplayer/asbplayer#439]
A feature to resume playback of an episode would be good because the tab sometimes reloads on mobile

**Comment by artjomsR** [comment]
Agree, this would be useful for longer content that isn't always feasible to watch in 1 sitting

**Comment by ShanaryS** [comment]
Implemented in 1.20 through #1077 and #1089
